stable/storageos-operator/templates/storageoscluster_cr.yaml

```diff
@@ -15,11 +15,6 @@ spec:
   k8sDistro: {{ .Values.k8sDistro }}
   {{- end }}
 
-  {{- if .Values.cluster.images.node.repository }}
-  images:
-    nodeContainer: "{{ .Values.cluster.images.node.repository }}:{{ .Values.cluster.images.node.tag }}"
-  {{- end }}
-
```

Comment:
Since the operator's deployment sets the node image, this is no longer needed.

Reply:
Do this mean removing also from the CRD?